examples/fabric-admin/device_manager/DeviceSynchronization.cpp

```diff
@@ -211,14 +207,26 @@ void DeviceSynchronizer::StartDeviceSynchronization(chip::Controller::DeviceCont
     MoveToState(State::Connecting);
 }
 
-void DeviceSynchronizer::GetUid(EndpointId remoteEndpointIdOfInterest)
+void DeviceSynchronizer::GetUniqueId()
 {
     VerifyOrDie(mState == State::ReceivedResponse);
     VerifyOrDie(mController);
+
+    auto * device = DeviceMgr().FindDeviceByNode(mCurrentDeviceData.node_id);
+    if (mCurrentDeviceData.has_unique_id || !device)
+    {
+        // We already have a UniqueId, or this device is not assoicated with a remote Fabric Sync Administrator,
```

Comment:
for debug purposes, some logs may be helpful here, reporting if device already has a unique id or if device has no remote fabric sync admin.

Reply:
Hmmm what do you think about only reporting if we have to fetch the UniqueID as the more common case is we don't need to do that

Reply:
My thinking was that I would like to be able to follow the code as I am reading logs. Log for teching unique id is fine, however I would also maybe want to differentiate between define _being nullptr vs device havin a unique_id already. If this can be done through other logs that is fine as well.